api.web.yaml

```diff
@@ -598,6 +598,7 @@ components:
         - title
         - data_controller_organisation
         - additional_study_admin_usernames
+        - approval_status
```

Comment:
I think this needs to be in the `Study` rather than `StudyBase` schema, as it's not settable in the request

Reply:
yeah - no idea. [upstream](https://github.com/oapi-codegen/oapi-codegen) weirdness

could consider splitting out the enum into its own `StudyApprovalStatus` schema for a (hopefully!) nicer generated name